src/js-marketing-notification.test.ts

```diff
@@ -1,76 +1,58 @@
 import { JsMarketingNotification, Notification } from './js-marketing-notification';
 import { LocalStorageManager } from './github/gtmsportswear/js-local-storage-manager@1.0.2/local-storage-manager';
 
-let notifier: JsMarketingNotification,
-    notification: Notification,
-    lsm;
+let lsm,
+    notifier: JsMarketingNotification,
+    notification: Notification = {
+      tabContent: {
+        node: document.createElement('h2')
```

Comment:
I think we could forgo the `node` property if we choose to remove the `classNames` property. The `tabContent` is currently an object with multiple properties to allow different data to be applied to the `tabContent` in the future if wanting to be used when creating the tab and body element.